### DISCUSSION TITLE
Does the Plex V5 script support discrete GPU Hardware Transcoding?

### DISCUSSION THREAD

**Discussion by sco0by2k** [tteck/Proxmox#1073] [Help Needed]
Hi, relative Proxmox/LXC noob here.

The Plex v5 script installs and runs with no issues, direct play works great but any transcoding is software only.

Should the script work with an Nvidia GPU by default or does the LXC need further configuration?

The node has 2 cards in it:

ls -l /dev/dri/
total 0
drwxrwxrwx 2 root root        120 Feb 18 00:12 by-path
crwxrwxrwx 1 root video  226,   0 Feb 18 00:12 card0
crwxrwxrwx 1 root video  226,   1 Feb 18 00:12 card1
crwxrwxrwx 1 root render 226, 128 Feb 18 00:12 renderD128
crw-rw---- 1 root render 226, 129 Feb 18 00:12 renderD129

I tried editing the .conf file from renderD128 to renderD129 but still software only.

Am I missing something obvious or is it just not a supported configuration?

TIA!

**Comment by BeerBelly** [comment]
It does, but depending on the generation of the processor you might need some extra steps. Which processor are you using?

**Reply by sco0by2k** [reply]
It's a Xeon E5-2680 V4 so it does not have any built-in video decoding at all as far as I am aware.

That was why I wanted to see if it would work with a discrete Nvidia GPU.
